### DISCUSSION TITLE
Allow singletons in m2a builder use

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#5083] [Feature Requests]
From https://github.com/directus/directus/issues/4406:

> [...] the idea of a page builder is that you can have that builder available on multiple pages. All I want to be able to accommodate is adding singleton components whose content doesn't change alongside other "dynamic" components whose contents change depending on their individual instances.

**Comment by tylerforesthauser** [comment]
This feature would also allow for "one-off" page-builder sections/components where you could create a single, unique page-builder section for just one particular page (rather than creating an entire collection that will only ever have just one entry).

**Comment by tylerforesthauser** [comment]
Also, I would like to get the One-to-Any phrase in here, since that's essentially what this would be (in my eyes, anyways).

Related: https://github.com/directus/directus/discussions/4749

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
